app/src/main/res/layout/activity_account_screen.xml

```diff
@@ -1,23 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/background_main"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior"
     tools:context=".AccountScreen">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:orientation="vertical">
 
+
         <RelativeLayout
             android:layout_width="match_parent"
             android:layout_height="250dp"
             android:background="@color/background_secondary"
             android:padding="1dp">
 
+            <androidx.appcompat.widget.Toolbar
+                android:id="@+id/toolbar"
+                android:layout_width="0dp"
+                android:layout_height="?attr/actionBarSize"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent">
+            </androidx.appcompat.widget.Toolbar>
```

Comment:
I suggest you wrap everything in the ScrollView in a ConstraintLayout and put the toolbar outside of the topmost LinearLayout so the toolbar visible. This suggestion also adds the app title that shows up in other screens.
Finally, close the new ConstraintLayout after the last LinearLayout.
```suggestion
       <androidx.constraintlayout.widget.ConstraintLayout
        android:layout_width="match_parent"
        android:layout_height="match_parent">

        <androidx.appcompat.widget.Toolbar
            android:id="@+id/toolbar"
            android:layout_width="0dp"
            android:layout_height="?attr/actionBarSize"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toTopOf="parent">
        </androidx.appcompat.widget.Toolbar>

        <TextView
            android:id="@+id/titleTextView"
            android:layout_width="156dp"
            android:layout_height="wrap_content"
            android:fontFamily="@font/bayon"
            android:text="@string/app_name"
            android:textAlignment="center"
            android:textColor="#ffffff"
            android:textSize="40sp"
            app:layout_constraintBottom_toBottomOf="@+id/toolbar"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintHorizontal_bias="0.498"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toTopOf="parent"
            app:layout_constraintVertical_bias="0.0" />
        <LinearLayout
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            app:layout_constraintTop_toBottomOf="@id/toolbar"
            app:layout_constraintStart_toStartOf="parent"
            android:orientation="vertical">


            <RelativeLayout
                android:layout_width="match_parent"
                android:layout_height="250dp"
                android:background="@color/background_secondary"
                android:padding="1dp">
```